features/areas.feature

```diff
@@ -1,366 +1,327 @@
-                  "variable": {
-                    "mapFrom": [],
-                    "name": "city"
-                  }
-                }
-              },
-              {
-                "node": {
-                  "code": "2",
-                  "label": "Belfast",
-                  "variable": {
-                    "mapFrom": [],
-                    "name": "city"
-                  }
-                }
+                ]
               }
-            ]
+            }
           }
         }
       }
     }
-  }
-}
-      """
-
-      And I GET "/areas"
-
+    """
+    And I GET "/areas"
     Then I should receive the following JSON response:
     """
     {
-    "areas": [
+      "areas": [
         {
-            "id": "E",
-            "label": "England",
-            "area-type": "country"
+          "id": "E",
+          "label": "England",
+          "area-type": "country"
         },
         {
-            "id": "N",
-            "label": "Northern Ireland",
-            "area-type": "country"
+          "id": "N",
+          "label": "Northern Ireland",
+          "area-type": "country"
         },
         {
-            "id": "0",
-            "label": "London",
-            "area-type": "city"
+          "id": "0",
+          "label": "London",
+          "area-type": "city"
         },
         {
-            "id": "1",
-            "label": "Liverpool",
-            "area-type": "city"
+          "id": "1",
+          "label": "Liverpool",
+          "area-type": "city"
         },
         {
-            "id": "2",
-            "label": "Belfast",
-            "area-type": "city"
+          "id": "2",
+          "label": "Belfast",
+          "area-type": "city"
         }
-    ]
-}
+      ]
+    }
     """
 
-    Scenario: Getting areas invalid dataset
-      When the following area query response is available from Cantabular api extension for the dataset "Test":
-      """
-      {
-  "data": {
-    "dataset": null
-  },
-  "errors": [
+  Scenario: Getting areas invalid dataset
+    When the following area query response is available from Cantabular api extension for the dataset "Test":
+    """
     {
-      "message": "404 Not Found: dataset not loaded in this server",
-      "locations": [
+      "data": {
+        "dataset": null
+      },
+      "errors": [
         {
-          "line": 2,
-          "column": 2
+          "message": "404 Not Found: dataset not loaded in this server",
+          "locations": [
+            {
+              "line": 2,
+              "column": 2
+            }
+          ],
+          "path": [
+            "dataset"
+          ]
         }
-      ],
-      "path": [
-        "dataset"
       ]
     }
-  ]
-}
-      """
+    """
     And I GET "/areas?dataset=Test"
-
     Then I should receive the following JSON response:
     """
     {
-    "errors": [
+      "errors": [
         "failed to get areas: error(s) returned by graphQL query"
```

Comment:
Some of my PR comments have expressed hesitation at leaking implementation details. They may be applicable here. 
Perhaps @ChrysmOre can clarify.

Reply:
Fair point, although I guess worth noting that this PR doesn't change the message/leak anything that wasn't being leaked before.

Reply:
true, but other than that LGTM.

Reply:
👍 Have removed the cantabular reference.

Reply:
okay looks good man